src/test/java/com/artipie/nuget/NugetITCase.java

```diff
@@ -139,15 +169,14 @@ private String exec(final String... command) throws Exception {
     }
 
     private String pushPackage() throws Exception {
-        final String uri = "http://host.testcontainers.internal:%d/my-nuget/index.json";
-        final String pckg = UUID.randomUUID().toString();
+        this.pckg = "newtonsoft.json";
         Files.write(
-            this.tmp.resolve(pckg),
+            this.tmp.resolve(this.pckg),
             new NewtonJsonResource("newtonsoft.json.12.0.3.nupkg").bytes()
         );
         return this.exec(
-            "dotnet", "nuget", "push", pckg, "--api-key", "this.source",
-            "-s", String.format(uri, this.port)
+            "dotnet", "nuget", "push", this.pckg,
+            "-s", String.format("%s/index.json", this.url)
```

Comment:
@genryxy file name you pass to `nuget push` command does not matter. NuGet repository extracts package name from it's content. So I'd suggest to keep using `UUID.randomUUID().toString()` for filename same as it was